src/Tizen.NUI/src/public/Accessibility/Accessibility.cs

```diff
@@ -226,10 +226,10 @@ public void SetHighlightFrameView(View view)
         public View GetCurrentlyHighlightedView()
         {
             var ptr = Interop.ControlDevel.DaliAccessibilityAccessibleGetCurrentlyHighlightedActor();
+
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
-            if (ptr == IntPtr.Zero)
-                return null;
-            return new View(ptr, true);
+
+            return this.GetInstanceSafely<View>(ptr);
```

Comment:
nice tip!

Reply:
How did you get this GetInstanceSafely from!! COOL!!